sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Action.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System;
+
+namespace Azure.Security.KeyVault.Certificates
+{
+    /// <summary>
+    /// Supported JsonWebKey key types (kty)
+    /// </summary>
+    public struct Action
+    {
+        private string _value;
+        internal const string AutoRenewValue = "AutoRenew";
+        internal const string EmailContactsValue = "EmailContacts";
+
+        public Action(string Action)
```

Comment:
Should be lowercase "action". Also, for consistency, do we want all these values throughout structs like this to just be "value"?